internal/generator/main.go

```diff
@@ -0,0 +1,96 @@
+package main
```

Comment:
```suggestion
package generator
```

and the file is `generator.go`